### ISSUE TITLE
Is Fatal Frame 3 supported?

### ISSUE THREAD

**Issue by Swagsy** [wagrenier/Obscura#13]
I've noticed there's no file table for Fatal Frame 3 in the ZeroFileDictionary.json, is a complete table entry the only thing needed left to get the extraction working?

**Comment by wagrenier** [comment]
It is planned and the code to support it is there. Also FF3 won't need entries in that JSON file because filenames are already contained within a file even on the release version.

**Comment by Swagsy** [comment]
Ah gotcha, was looking to extract Rei from FF3 eventually so that sounds awesome, hope it'll be possible eventually with Obscura. Thank you for the tool, really useful 🙂

**Comment by wagrenier** [comment]
If you just want to extract the files you can use this tool here: https://github.com/kapdap/ZeroUnpacker. Take note that I am not related to this tool whatsoever and you will need to compile it yourself. If you have the knowledge it should not be too hard to do. Otherwise, just wait for us to add FF3 support :)

**Comment by Swagsy** [comment]
Yeah I actually saw that one as well, seemed like it's only for unpacking the raw files and doesn't have a way to convert the models through unfortunately, unless it's possible I can use the unpacked files with this tool's converter?

**Comment by wagrenier** [comment]
You can extract them and some models may work but I expect most won't because the game changed it's model archive format from pk2 to pk3 and right now Obscura does not support pk3. However, the model files are still in the same format of sgd version 0x1050.
